docs/src/main/paradox/r2dbc.md

```diff
@@ -56,11 +56,15 @@ YugaByte
 :  @@snip [YugaByte Schema](/akka-projection-r2dbc/ddl-scripts/create_tables_yugabyte.sql)
 
 H2
-:  @@snip [H2 Schema](/akka-projection-r2dbc/src/it/resources/application-h2.conf) { #schema }
+:  @@snip [H2 Schema](/akka-projection-r2dbc/src/main/resources/h2-default-projection-schema.conf) { #schema }
```

Comment:
the previous snippet showed this part:
```
akka.persistence.r2dbc.connection-factory {
  additional-init = """
```

That was quite useful, could we somehow make it show? Does hcon support `/*` comments?

```
akka.projection.r2dbc.default-h2-schema =
/*
// #schema
akka.persistence.r2dbc.connection-factory {
  additional-init = """
// #schema
*/
  """
// #schema
  CREATE TABLE
```

Reply:
No support for multi line comments in hocon, but I figured out some other trixery to do it.